README.md

```diff
@@ -20,53 +20,35 @@ Contributing to FoundationDB can be in contributions to the code base, sharing y
 
 ### Binary downloads
 
-Developers interested in using the FoundationDB store for an application can get started easily by downloading and installing a binary package. Please see the [downloads page](https://www.foundationdb.org/download/) for a list of available packages.
+Developers interested in using FoundationDB can get started by downloading and installing a binary package. Please see the [downloads page](https://www.foundationdb.org/download/) for a list of available packages.
 
 
 ### Compiling from source
 
 Developers on an OS for which there is no binary package, or who would like
 to start hacking on the code, can get started by compiling from source.
 
-Currently there are two build systems: a collection of Makefiles and a
-CMake-based build system. Both of them should currently work for most users,
-and CMake should be the preferred choice as it will eventually become the only
-build system available.
+The official docker image for building is `foundationdb/foundationdb-build`. It has all dependencies installed. Use `-DBOOST_ROOT=/opt/boost_1_72_0/` in your cmake command. To build outside the official docker image you'll need at least these dependencies:
+
+1. Install cmake Version 3.13 or higher [CMake](https://cmake.org/)
+1. Download version 1.72 of [Boost](https://dl.bintray.com/boostorg/release/1.72.0/source/boost_1_72_0.tar.bz2)
+1. Unpack boost (you don't need to compile it)
```

Comment:
I would remove those or at least mention that these steps are optional. For someone who just wants to build fdb once, the current process of downloading boost through cmake is good enough

Reply:
Removed